src/njs_cnst.h

```diff
@@ -0,0 +1,555 @@
+        njs_value_t _setUTCMilliseconds;
+        njs_value_t _setUTCMinutes;
+        njs_value_t _setUTCMonth;
+        njs_value_t _setUTCSeconds;
+        njs_value_t _shift;
+        njs_value_t _sign;
+        njs_value_t _sin;
+        njs_value_t _sinh;
+        njs_value_t _size;
+        njs_value_t _slice;
+        njs_value_t _some;
+        njs_value_t _sort;
+        njs_value_t _source;
+        njs_value_t _species;
+        njs_value_t _splice;
+        njs_value_t _split;
+        njs_value_t _sqrt;
+        njs_value_t _stack;
+        njs_value_t _startsWith;
+        njs_value_t _status;
+        njs_value_t _sticky;
+        njs_value_t _stream;
+        njs_value_t _string;
+        njs_value_t _stringify;
+        njs_value_t _subarray;
+        njs_value_t _substr;
+        njs_value_t _substring;
+        njs_value_t _symbol;
+        njs_value_t _swap16;
+        njs_value_t _swap32;
+        njs_value_t _swap64;
+        njs_value_t _tan;
+        njs_value_t _tanh;
+        njs_value_t _test;
+        njs_value_t _then;
+        njs_value_t _times;
+        njs_value_t _toDateString;
+        njs_value_t _toExponential;
+        njs_value_t _toFixed;
+        njs_value_t _toISOString;
+        njs_value_t _toJSON;
+        njs_value_t _toLocaleDateString;
+        njs_value_t _toLocaleString;
+        njs_value_t _toLocaleTimeString;
+        njs_value_t _toLowerCase;
+        njs_value_t _toPrimitive;
+        njs_value_t _toPrecision;
+        njs_value_t _toReversed;
+        njs_value_t _toSorted;
+        njs_value_t _toSpliced;
+        njs_value_t _toStringTag;
+        njs_value_t _toString;
+        njs_value_t _toTimeString;
+        njs_value_t _toUpperCase;
+        njs_value_t _toUTCString;
+        njs_value_t _trimEnd;
+        njs_value_t _trimStart;
+        njs_value_t _trim;
+        njs_value_t _trunc;
+        njs_value_t _type;
+        njs_value_t _usec;
+        njs_value_t _unscopables;
+        njs_value_t _unshift;
+        njs_value_t _utf_8;
+        njs_value_t _value;
+        njs_value_t _valueOf;
+        njs_value_t _values;
+        njs_value_t _version;
+        njs_value_t _version_number;
+        njs_value_t _write;
+        njs_value_t _writable;
+        njs_value_t _writeDoubleBE;
+        njs_value_t _writeDoubleLE;
+        njs_value_t _writeFloatBE;
+        njs_value_t _writeFloatLE;
+        njs_value_t _writeIntBE;
+        njs_value_t _writeIntLE;
+        njs_value_t _writeInt16BE;
+        njs_value_t _writeInt16LE;
+        njs_value_t _writeInt32BE;
+        njs_value_t _writeInt32LE;
+        njs_value_t _writeInt8;
+        njs_value_t _writeUIntBE;
+        njs_value_t _writeUIntLE;
+        njs_value_t _writeUInt16BE;
+        njs_value_t _writeUInt16LE;
+        njs_value_t _writeUInt32BE;
+        njs_value_t _writeUInt32LE;
+        njs_value_t _writeUInt8;
+        njs_value_t _written;
+   } vs;
+};
+
+typedef union njs_cnst_values_s  njs_cnst_values_t;
```

Comment:
We do not need to split the typedef from a declaration if it is not a recursive struct.
So in can be declared as `typedef struct { ..} njs_predefined_values_t`.